synapse/lib/layer.py

```diff
@@ -208,18 +208,24 @@ async def getIden(self):
 
 STOR_FLAG_ARRAY = 0x8000
 
-EDIT_NODE_ADD = 0     # (<type>, (<valu>, <type>), ())
-EDIT_NODE_DEL = 1     # (<type>, (<valu>, <type>), ())
-EDIT_PROP_SET = 2     # (<type>, (<prop>, <valu>, <oldv>, <type>), ())
-EDIT_PROP_DEL = 3     # (<type>, (<prop>, <oldv>, <type>), ())
-EDIT_TAG_SET = 4      # (<type>, (<tag>, <valu>, <oldv>), ())
-EDIT_TAG_DEL = 5      # (<type>, (<tag>, <oldv>), ())
-EDIT_TAGPROP_SET = 6  # (<type>, (<tag>, <prop>, <valu>, <oldv>, <type>), ())
-EDIT_TAGPROP_DEL = 7  # (<type>, (<tag>, <prop>, <oldv>, <type>), ())
-EDIT_NODEDATA_SET = 8 # (<type>, (<name>, <valu>, <oldv>), ())
-EDIT_NODEDATA_DEL = 9 # (<type>, (<name>, <valu>), ())
-EDIT_EDGE_ADD = 10    # (<type>, (<verb>, <destnodeiden>), ())
-EDIT_EDGE_DEL = 11    # (<type>, (<verb>, <destnodeiden>), ())
+# Edit types (etyp)
+
+EDIT_NODE_ADD = 0     # (<etyp>, (<valu>, <type>), ())
+EDIT_NODE_DEL = 1     # (<etyp>, (<oldv>, <type>), ())
+EDIT_PROP_SET = 2     # (<etyp>, (<prop>, <valu>, <oldv>, <type>), ())
+EDIT_PROP_DEL = 3     # (<etyp>, (<prop>, <oldv>, <type>), ())
+EDIT_TAG_SET = 4      # (<etyp>, (<tag>, <valu>, <oldv>), ())
+EDIT_TAG_DEL = 5      # (<etyp>, (<tag>, <oldv>), ())
+EDIT_TAGPROP_SET = 6  # (<etyp>, (<tag>, <prop>, <valu>, <oldv>, <type>), ())
+EDIT_TAGPROP_DEL = 7  # (<etyp>, (<tag>, <prop>, <oldv>, <type>), ())
+EDIT_NODEDATA_SET = 8 # (<etyp>, (<name>, <valu>, <oldv>), ())
+EDIT_NODEDATA_DEL = 9 # (<etyp>, (<name>, <oldv>), ())
+EDIT_EDGE_ADD = 10    # (<etyp>, (<verb>, <destnodeiden>), ())
+EDIT_EDGE_DEL = 11    # (<etyp>, (<verb>, <destnodeiden>), ())
+
+EDIT_LAYR_ADD = 100   # (used by cortex)
```

Comment:
This seems like it's creeping into being something that we should have enveloping take care of rather than having mixed messages.